### DISCUSSION TITLE
Reducing the number of files in a plotfile directory

### DISCUSSION THREAD

**Discussion by wjwhit** [AMReX-Codes/amrex#3954] [Q&A] [answered]
Greetings, 

I am currently running an AMReX based code on an HPC system that has a relatively strict limitation on the number of files in my run directory - the size of the files is not an issue. Is there any way to reduce the number of files that compose each plotfile directory, ideally during runtime? 

Thank you,
William

**Comment by WeiqunZhang** [comment, marked as answer]
Call `amrex::VisMF::SetNOutFiles(n)`, where `n` can be say 16. The default in amrex is 256.

**Reply by wjwhit** [reply]
Perfect, thank you!
